src/octoprint/static/js/app/viewmodels/control.js

```diff
@@ -765,6 +782,37 @@ $(function () {
             self.webcamHlsEnabled(false);
             self.webcamWebRTCEnabled(true);
         };
+
+        self._updateVideoTagWebcamLayout = function (containerId) {
```

Comment:
This needs more comments tbh. I see myself looking at this code in half a year and having NO idea anymore what the sections are supposed to achieve.

Reply:
Will do :)

Reply:
Added